training/refit/__init__.py

```diff
@@ -97,7 +98,21 @@ def save(self,
     def __get_file_path(self, object_name: str):
         return f"import/{self.project_guid}/{object_name}"
 
-    def import_file(self,
+    def import_data(self,
+                dataframe: pd.DataFrame(),
+                object_name: str) -> str:
```

Comment:
the only parameter to this function will only be the dataframe, also `pd.DataFrame()` is not the correct way to do type hinting I believe. Lets try to make the function signature look like this and generate object_name by generating a new GUID or something. 

```suggestion
    def import_data(self,
                dataframe: pd.DataFrame) -> str:
```